amqp/queue/rabbitmq.go

```diff
@@ -34,6 +34,25 @@ func (d *RabbitMQDelivery) Nack(multiple, requeue bool) error {
 	return d.Delivery.Nack(multiple, requeue)
 }
 
+func (d *RabbitMQDelivery) SetHeader(key, value string) error {
```

Comment:
use interface type instead?

Reply:
@hannes-smspark Fixed, now interface{} is used in the API.